### ISSUE TITLE
Integration with struct plugin

### ISSUE THREAD

**Issue by Juergen-aus-Koeln** [giterlizzi/dokuwiki-template-bootstrap3#510]
### Description

When editing a page with "struct data" the area to edit this data is beneath the page content. And this area normally can be folded in and out pressing the headline of the schema. This is especially helpful if you have several schemas on one page.

### Steps to reproduce

1. Edit a page with struct data
2. Attempt to click on the headline of the schema

**Expected behavior:** 
The lines where the struct data can be edited are clapped together (folded) or opened again

**Actual behavior:** 
nothing happens

### Versions

* Template v2020-08-28 (stable/20200828)
* Release 2020-07-29 "Hogfather"
* PHP-Version: 7.4.8
* Firefox 80.0.1 (64-Bit) / Chrome Version 85.0.4183.83 (Offizieller Build) (64-Bit)

### Screenshots or Logs

![2020-09-03 17_22_22-✎ 14 Bedienung (ECX36 01)  TEST-WIKI](https://user-images.githubusercontent.com/1800472/92134846-4ef53d80-ee0a-11ea-82cf-2a4f63e1aab2.png)

and here the same after clicking the headline in the old "Greebo":

![2020-09-03 17_25_31-✎ 14 Bedienung (H3E 01)  Schütte-Wiki](https://user-images.githubusercontent.com/1800472/92135094-a1cef500-ee0a-11ea-8f4f-171c1f4a6c79.png)

**Comment by giterlizzi** [comment]
You have errors in JS Console ?

**Comment by Juergen-aus-Zuendorf** [comment]
Yes, I've found these error messages in the console of firefox which refer to bootstrap:

a) view modus:
```
Uncaught TypeError: e.indexOf is not a function
    load jQuery
    <anonymous> http://localhost/wiki/lib/exe/js.php?t=bootstrap3&tseed=133fe3c3f085e6c1d9c0ba9c749b4df9:1
jquery.php:2:84831
Uncaught TypeError: IndexmenuUtils is undefined
    dTree http://localhost/wiki/lib/exe/js.php?t=bootstrap3&tseed=133fe3c3f085e6c1d9c0ba9c749b4df9:1
    <anonymous> http://localhost/wiki/doku.php?id=redaktionssystem:ccms:meta:dps00:m0000003.007.v01:332
```
b) edit modus:
```
Uncaught TypeError: e.indexOf is not a function
    load jQuery
    <anonymous> http://localhost/wiki/lib/exe/js.php?t=bootstrap3&tseed=133fe3c3f085e6c1d9c0ba9c749b4df9:1
```
Regards
Juergen
